src/scss/_index_sections.scss

```diff
@@ -76,6 +76,15 @@
       font-size: 1.2em;
     }
 
+    img {
+      width: 100%;
```

Comment:
This should be `max-width` and `max-height`. Otherwise, if we have a tiny image in the announcement it will be always scaled to the full width.

E.g. if I replace the image with the Nucleus icon:

![screenshot_20180405_113659](https://user-images.githubusercontent.com/3035868/38358526-c13556c0-38c5-11e8-8944-9a4f5d19beef.png)

Reply:
Yeah sorry, I should've realized that.